digiwf-integrations/digiwf-ticket-integration/digiwf-ticket-integration-core/src/main/java/de/muenchen/oss/digiwf/ticket/integration/adapter/out/zammad/ZammadAdapter.java

```diff
@@ -17,22 +22,36 @@ public class ZammadAdapter implements TicketOutPort {
 
 
     @Override
-    public void updateTicket(String ticketId, Article article, TicketStatus status) {
+    public void updateTicket(String ticketId, Article article, TicketStatus status, List<FileContent> attachments) {
 
-        final var ticketUpdateDto = mapToDTO(ticketId, article, status);
+        final var ticketUpdateDto = mapToDTO(ticketId, article, status, attachments);
 
         ticketsApi.updateTicket(ticketId, ticketUpdateDto, null, article.getUserId()).block();
 
     }
 
-    private UpdateTicketDTO mapToDTO(String ticketId, Article article, TicketStatus status) {
+    private UpdateTicketDTO mapToDTO(String ticketId, Article article, TicketStatus status, List<FileContent> attachments) {
         val ticketUpdateDto = new UpdateTicketDTO();
         // Note: TicketId is required in body and url
         ticketUpdateDto.setId(ticketId);
         ticketUpdateDto.setState(mapStatus(status));
```

Comment:
Theoretisch gilt folgendes: `"When using state 'pending reminder' or 'pending action' you have to set 'pending_time', too.");`
Noch in Klärung ob dann `pending_time` eingebaut oder nur `open` und `closed` gültig ist.

Reply:
`PENDING_CLOSE` und `PENDING_REMINDER` wurden entfernt, da kein sinnvoller UseCase ersichtlich